gems/smithy/lib/smithy/anvil/client/views/plugins/endpoint.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Anvil
+    module Client
+      module Views
```

Comment:
We should think about using a compact namespace Anvil::Client::Views or just dropping Views (and removing views folder) as Anvil is already the designated namespace for code writing.

Reply:
Yeah, I agree this namespace is getting long.  I can do that refactor outside of this PR.